src/layout/MainContent.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { Box } from "@mui/material";
+
+const MainContent: React.FC = () => {
+  return (
+    <Box
+      sx={{
```

Comment:
Avoid using inline styles

Reply:
got it dai, but yeha nir just sections visualize grn ko lagi matra temporarily haleko dummy code ho yo so yesma use gareko... sections kko code change hunx aba auta auta garer